odoo/http.py

```diff
@@ -501,6 +647,9 @@ def fake_start_response(status, headers):
         current_thread.perf_t0 = time.time()
         current_thread.url = httprequest.url
 
+        if self.statics is None:
+            self.load_statics()
```

Comment:
Because the method populates `addons_manifest`. I'm working on isolating `http.addons_manifest` elsewhere so I can refactor this part to be a lazy_property.